blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/RollupRunnable.java

```diff
@@ -59,13 +62,14 @@
         }
     }
 
-    public RollupRunnable(RollupExecutionContext executionContext, SingleRollupReadContext singleRollupReadContext, RollupBatchWriter rollupBatchWriter) {
+    public RollupRunnable(RollupExecutionContext executionContext, SingleRollupReadContext singleRollupReadContext, RollupBatchWriter rollupBatchWriter, ThreadPoolExecutor enumValidatorExecutor) {
```

Comment:
I noticed that child classes are calling this constructor with the ``ThreadPoolExecutor enumValidatorExecutor`` parameter taking a ``null``.  I assumed because the executor isn't used as the ``run()`` method is overridden.  Do these classes need to subclass this class?  Would it be clearer to have this and its sub-classes all implement a common interface rather than sub-class ``RollupRunnable``?  Just curious.